play-framework/websockets/app/utils/MessageConverter.java

```diff
@@ -0,0 +1,34 @@
+package utils;
+
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import dto.MessageDTO;
+import dto.RequestDTO;
+
+/**
+ * Created by alfred on 07 Jun 2019
+ */
+public class MessageConverter {
+    public static MessageDTO jsonNodeToMessage(JsonNode jsonNode) {
+        ObjectMapper mapper = new ObjectMapper();
+        final MessageDTO messageDTO = mapper.convertValue(jsonNode, MessageDTO.class);
+        return messageDTO;
+    }
+
+    public static JsonNode messageToJsonNode(MessageDTO messageDTO) {
+        ObjectMapper mapper = new ObjectMapper();
+        JsonNode jsonData = mapper.convertValue(messageDTO, JsonNode.class);
+        return jsonData;
+    }
+    public static RequestDTO jsonNodeToRequest(JsonNode jsonNode) {
+        ObjectMapper mapper = new ObjectMapper();
```

Comment:
FWIW this is not the ideal way to use `ObjectMapper`. You don't have to change it, but it's common not to keep creating new instances, and use a `private static final ObjectMapper OBJECT_MAPPER` constant

Reply:
I have changed to use the private static final constant.